editions/tw5.com/tiddlers/demonstrations/NestedLists.tid

```diff
@@ -0,0 +1,44 @@
+title: NestedLists
+tags: ListWidget
+created: 20150106180000000
+modified: 20150106180000000
+
+The following sidebar tabs give examples of grouped lists created by nesting.
+
+!! Types Tab
+
+The list in the [[Types|$:/core/ui/MoreSideBar/Types]] tab is generated with this markup:
+
+```
+<$list filter="[!is[system]has[type]each[type]sort[type]]">
```

Comment:
Ok to add that to...

**$:/editions/tw5.com/wikitext-macros**

?

How about...

```
<<tw-demo>>
<<tw-demo-html>>
<<tw-code>>
<<tw-code-demo>>
```

?